docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_CRNN_From_Tensorflow.md

```diff
@@ -3,54 +3,50 @@
 This tutorial explains how to convert a CRNN model to Intermediate Representation (IR).
 
 There are several public versions of TensorFlow CRNN model implementation available on GitHub. This tutorial explains how to convert the model from
-the [CRNN Tensorflow](https://github.com/MaybeShewill-CV/CRNN_Tensorflow) repository to IR. 
+the [CRNN Tensorflow](https://github.com/MaybeShewill-CV/CRNN_Tensorflow) repository to IR, which was validated with python 3.7, tensorflow 1.15.0, protobuf 3.19.0.
 If you have another implementation of CRNN model, it can be converted to OpenVINO IR in a similar way. You need to get inference graph and run Model Optimizer on it.
 
 **To convert this model to the IR:**
 
 **Step 1.** Clone this GitHub repository and checkout the commit:
```

Comment:
```suggestion
**Step 1.** Clone this GitHub repository and check out the commit:
```

Reply:
`checkout` is specific git option, please see example in another doc: https://docs.openvino.ai/latest/openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_EfficientDet_Models.html#getting-a-frozen-tensorflow-model. I suggest to keep as it is.

Reply:
Understand, but a command cannot be used as a verb if it's a noun. Using checkout in this phrase should be incorrect. :)

Reply:
You can also visit https://git-scm.com/docs/git-checkout/ to see the difference between them.

Reply:
Thanks for your clarification, update doc according to your comment.